canonical-websites/www.ubuntu.com
# Update sever management to server plans and pricing

## Done

* renamed Server management, Server plans and pricing
* added shared tables from /support/plans-and-pricing
* added maas to top section
* removed unused partials
* updated the [management copy doc](https://docs.google.com/document/d/1Oqzs4utcIAdmFwZlSqlA6m5ioDlNnAjj_OTi_RHF2f8/edit#)
* updated the [hyperscale copy doc](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/server/plans-and-pricing)
- see that the title has changed and matches the [copy doc](/server/plans-and-pricing)
- see that [hyperscale](http://0.0.0.0:8001/server/hyperscale) links to /server/plans-and-pricing
- try http://0.0.0.0:8001/server/management and see that it redirect to http://0.0.0.0:8001/server/plans-and-pricing
